.buildkite/e2e-pipeline.yml

```diff
@@ -1 +1,30 @@
-# Intentional empty definition because it is in 8.x
\ No newline at end of file
+- label: "E2E pipeline"
```

Comment:
similar to build CI, E2E CI jobs run when PR gets merged on a specific branch.
FYI: periodic E2E job runs every branch, scripts will be placed on plugin main branch.